### PR TITLE
fix #19940. Added common steps of plugin translation

### LINKED ISSUE #19940: Update Internationalization (i18n) contributor docs
The current [Internationalization documentation page](https://github.com/WordPress/gutenberg/blob/update/docs-contrib-dev/docs/contributors/localizing.md) in the Contributor Guide needs some updating.

I need help updating the page, I'm confused by the current phrasing and don't know enough what the proper process is to update myself. My guess is it targeted at someone familiar with how WordPress translations already work, which I am not.

I think we need the page to be written for someone who might be new to WordPress and wants to help translate Gutenberg. Information doesn't have to be duplicated if it already exists elsewhere in WordPress Handbooks, just need good pointers and guidance for someone new to get started.

### PR DESCRIPTION
**Pull request by atachibana**
## Description

This PR will fix issue #19940 of @mkaz. 
For new translators, added general description about WordPress Translation and link to Polyglots handbook.

## How has this been tested?

Spell and grammar were checked on https://www.online-spellcheck.com/result/ba540f6ce8ca06822999f9a1de3c99a599bbe87b
Two links are reachable.

## Screenshots 
n/a

## Types of changes

Added the 1st paragraph.

## Checklist:
- [n/a] My code is tested.
- [n/a] My code follows the WordPress code style. 
- [n/a] My code follows the accessibility standards. 
- [n/a] My code has proper inline documentation. 
- [n/a] I've included developer documentation if appropriate. 
- [n/a] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Review by mkaz — APPROVED** [review]
Thanks for updating and the link to more information.

File: docs/contributors/localizing.md
Comment (aristath) [inline docs/contributors/localizing.md:4]: Just some grammar fixes before we can merge this:
```suggestion
As general WordPress Plugins, Gutenberg Plugin is translated via a translation management system at https://translate.wordpress.org. If you are not familiar with this system and translation process, refer to [this document](https://make.wordpress.org/polyglots/handbook/tools/glotpress-translate-wordpress-org/) first.
```

**Review by aristath — CHANGES_REQUESTED** [review]

File: docs/contributors/localizing.md
Comment (aristath) [inline docs/contributors/localizing.md]: I'm not a native English speaker, but I've got the impression there are some articles & prepositions missing here. A native English speaker should review this before we merge it...

Comment (mkaz) [inline docs/contributors/localizing.md]:
```suggestion
The Gutenberg plugin is translated via the general plugin translation system (GlotPress) at https://translate.wordpress.org. Review the [GlotPress translation process documentation] (https://make.wordpress.org/polyglots/handbook/tools/glotpress-translate-wordpress-org/) for additional information.
```

**Review by aristath — APPROVED** [review]

**Comment by aristath** [comment]
The failing test is not related to this PR, I'll go ahead and merge it.

**Comment by talldan** [comment]
@mkaz @aristath Please could you make sure PRs have appropriate titles and labels when reviewing or merging. Thanks.

**Pull request merged**
